packages/server-core/src/user/avatar/avatar.hooks.ts

```diff
@@ -155,6 +155,28 @@ const updateUserAvatars = async (context: HookContext<AvatarService>) => {
   }
 }
 
+const addAndSortByUserName = async (context: HookContext<AvatarService>) => {
+  if (!context.params.query) return
+
+  const userNameAlias = 'userName'
+
+  const sort = context.params.query.$sort?.['owner']
+  delete context.params.query.$sort?.['owner']
+  const nameLike = context.params.query.name as any
+  delete context.params.query.name
+
+  const query = context.service.createQuery(context.params)
+
+  query.leftJoin(userPath, `${userPath}.id`, `${avatarPath}.userId`)
+  if (nameLike) {
+    query.where(`${avatarPath}.name`, 'LIKE', nameLike.$like)
```

Comment:
Why are we handling this ourself? I think knex will do it for us if we dont delete query.name params